components/x-podcast-launchers/storybook/index.jsx

```diff
@@ -1,4 +1,4 @@
-const { PodcastLaunchers } = require('../dist/PodcastLaunchers.cjs')
+import { PodcastLaunchers } from '../src/PodcastLaunchers'
 const { brand } = require('@financial-times/n-concept-ids')
```

Comment:
non-merge-blocking, is there any reason to keep these as `require` or should we standardise on `import`? if there's a reason it should probably be documented or commented?

Reply:
nope, there is no particular reason, at least for this case... it's leftover pre-migration code, I'll push the change